samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazorML.ConsoleApp/ModelBuilder.cs

```diff
@@ -83,13 +73,13 @@ private static async Task DownloadData()
         public static IEstimator<ITransformer> BuildTrainingPipeline(MLContext mlContext)
         {
             // Data process configuration with pipeline data transformations 
-            var dataProcessPipeline = mlContext.Transforms.Text.FeaturizeText("Comment_tf", "Comment")
-                                      .Append(mlContext.Transforms.CopyColumns("Features", "Comment_tf"))
+            var dataProcessPipeline = mlContext.Transforms.Text.FeaturizeText("SentimentText_tf", "SentimentText")
+                                      .Append(mlContext.Transforms.CopyColumns("Features", "SentimentText_tf"))
                                       .Append(mlContext.Transforms.NormalizeMinMax("Features", "Features"))
                                       .AppendCacheCheckpoint(mlContext);
 
             // Set the training algorithm 
-            var trainer = mlContext.BinaryClassification.Trainers.SdcaLogisticRegression(labelColumnName: "Sentiment", featureColumnName: "Features");
+            var trainer = mlContext.BinaryClassification.Trainers.AveragedPerceptron(labelColumnName: "Sentiment", numberOfIterations: 10, featureColumnName: "Features");
```

Comment:
Good change. AP w/ 10 iterations is the winning trainer we have for general text classification. We've benchmarked this combination rather throughly.

Reply:
That's a kudos to the AutoML / Model Builder teams since this is part of the code-gen. I take no credit for that 🙂